docs/guide/config_back_office.md

```diff
@@ -196,3 +196,27 @@ ezplatform:
                     additional_icons: /assets/images/icons/additional_icons.svg
                 default_icon_set: my_icons
 ```
+
+## Enable Commerce features
+
+Commerce features setting `commerce.enabled` by default is set to `false`. 
```

Comment:
How about:
"The `commerce.enabled` setting which controls, for example, X, Y, Z, is set to `false` by default"
so users have an idea what "commerce features" you refer to.